napari/utils/events/event.py

```diff
@@ -410,7 +410,7 @@ def connect(
         self,
         callback: Union[Callback, CallbackRef, CallbackStr, 'EventEmitter'],
         ref: Union[bool, str] = False,
-        position: Literal['first', 'last'] = 'first',
+        position: Literal['first', 'last'] = 'last',
```

Comment:
This is a pretty big change right? It means every event order in the whole codebase and plugins will be swapped? Are we sure we want to change the default here?

Reply:
Yes. After this change, events will be executed in connection order, not in reversed order. So callbacks from ViewerModel will be executed before callbacks from QtViewer

Reply:
I think in principle this is how I would expect it to work, but it's likely to affect things... I'm surprised tests are all passing xD

Reply:
Because the callbacks should be independent. 
But my investigation in original bug show that sometimes, during processing of callbacks, some QtLoop event processing happens and reval some problem. 

Also reported problem may happen only when using 3d view.

Reply:
Ok. I would ask some folks with complex plugins to try out this change before merging! I'll try with blik later.

Reply:
good point, I will try with napari-spatialdata. One thing @Czaki, correct me if I am wrong, this does not affect the order of events being emitted by QtViewer right (asking because of differences between windows / linux and macs here)? Just solely the viewer model first and then the QtViewer.

Reply:
`QtViewer` events are Qt events, so there should be no impact.

Reply:
> Ok. I would ask some folks with complex plugins to try out this change before merging! I'll try with blik later.

All seems to be working well on my side!